the commit puts it into default config- we put it here to activate it immediately
diff --git a/abuild.in b/abuild.in
index 76cd77b..bd38003 100644
--- a/abuild.in
+++ b/abuild.in
@@ -462,7 +462,7 @@ initdcheck() {
 	local i line
 	for i in $source; do
 		case $i in
-		*.initd) 
+		*.initd)
 			line=$(head -n 1 "$srcdir"/"$(filename_from_uri $i)")
 			;;
 		*) continue ;;
@@ -2793,6 +2793,8 @@ fi
 # for recursive action
 export REPODEST SRCDEST
 
+export GOFLAGS="$GOFLAGS -modcacherw"
+
 # add dbg subpackage if its enabled globally
 if [ -n "$DEFAULT_DBG" ] && ! subpackage_types_has "dbg" && ! options_has "!dbg" && [ "$arch" != "noarch" ]; then
         subpackages="$pkgname-dbg $subpackages"
